src/components/SearchNetwork/Search.styles.js

```diff
@@ -0,0 +1,57 @@
+import { fade, makeStyles } from '@material-ui/core/styles';
```

Comment:
This is going to make cleaning up the styling so much easier! thanks for adding this style sheet